android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/GpsFactory.java

```diff
@@ -236,6 +237,10 @@ public void attachLayout(Context context, @NonNull JSONObject jsonObject, @NonNu
             if (coordinateElements.length > 1) {
                 latitude = coordinateElements[0];
                 longitude = coordinateElements[1];
+                if (coordinateElements.length >= 4) {
+                    altitude = coordinateElements[2];
+                    accuracy = coordinateElements[3];
+                }
```

Comment:
Like the previous [pr](https://github.com/OpenSRP/opensrp-client-core/pull/705#discussion_r532604533) we can use an array to remove the need for the length checks.